deeplake/experimental/dataloader.py

```diff
@@ -110,15 +110,19 @@ def shuffle(self, shuffle: bool = True, buffer_size: int = 2048):
 
         Raises:
             ValueError: If .shuffle() has already been called.
+            ValueError: If dataset is view and shuffle is True
```

Comment:
Are we sure that indexes are tuple only in case if it is view?

Reply:
It also can be slice, but slice is ok to be shuffled.